controller/src/main/java/com/epam/esm/controller/CertificateController.java

```diff
@@ -153,11 +149,11 @@ public List<CertificateDto> searchCertificates(@RequestParam("part") String part
     public List<CertificateDto> sort(@RequestParam(value = "param_1", required = false) String name,
                                      @RequestParam(value = "param_2", required = false) String date){
         List<CertificateDto> certificateDtoList;
-        if(name.equals("name") && date.equals("date")){
+        if(name != null && date != null && name.equals("name") && date.equals("date")){
```

Comment:
Awful statement - just write "name".equals(name) and so on

Reply:
I got it, everything will be fixed